source/_components/device_tracker.tesla.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: "Tesla"
+description: "Instructions for how to integrate Tesla into Home Assistant."
+date: 2017-08-02 12:20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Presence Detection
+ha_release: 0.53
+---
+
+The `Tesla` platform allows you to get data about location of your [Tesla](https://www.tesla.com/) car within Home Assistant.
```

Comment:
**Spelling & Grammar**:
- Missing determiner for `location` -> `get data about the location`